server/views/pages/overview/partials/supportNeedsTabContents.test.ts

```diff
@@ -95,3 +76,55 @@ describe('Support Needs tab view', () => {
     expect($('h2').text()).toEqual('Sorry, the Curious service is currently unavailable.')
   })
 })
+
+declare module 'cheerio' {
+  interface Cheerio<T> {
+    heading(this: Cheerio<T>): Cheerio<T>
+    rapidAssessmentDate(this: Cheerio<T>): Cheerio<T>
+    inDepthAssessmentDate(this: Cheerio<T>): Cheerio<T>
+    primaryLddAndHealthNeeds(this: Cheerio<T>): Cheerio<T>
+    additionalLddAndHealthNeeds(this: Cheerio<T>): Cheerio<T>
+    selfDeclaredNeurodiversityDiagnosis(this: Cheerio<T>): Cheerio<T>
+    assessedNeurodiversityDiagnosis(this: Cheerio<T>): Cheerio<T>
+    neurodiversitySupportNeeds(this: Cheerio<T>): Cheerio<T>
+  }
+}
+
+const setupCheerioExtensionFunctions = ($: CheerioAPI) => {
+  // eslint-disable-next-line no-param-reassign
+  $.prototype.heading = function heading(): Cheerio<never> {
```

Comment:
... and then you implement those new functions on the prototype (the 'prototype' is javascript's term for what is essentially the class definition .... though of course javascript does not have classes! 🤣 )